packages/create-cloudflare/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "create-cloudflare",
-	"version": "2.13.2",
+	"version": "2.14.0",
```

Comment:
I guess I must have misread the convo from chat.

When I updated the changeset to include `feature` I also changed it from `patch` to `minor`. Someone asked if the version would also get automatically changed by `changesets`. My guess was that it wasn't. Someone else said editing these files is fine. I took that to mean it's ok to manually bump the version in `package.json` so that it's in sync with `minor`.

Is it the case that `changesets` will update the version number here even if I've changed from `patch` to `minor` in this PR?

Reply:
Yeah changesets updates your package.json based on the content of your changeset `.md` files at the time of merging the "Version packages" PR - you should be able to remove this package.json change, and just keep the changeset change, and you'll see the Version packages PR update itself